### DISCUSSION TITLE
What to store to Data Base and with what i can compair to take attendance?

### DISCUSSION THREAD

**Discussion by kd1-80587** [luicfrr/react-native-vision-camera-face-detector#92] [Q&A]
the output of this face detection gives json object as LOG  faces detected [{"bounds": {"height": 316, "width": 315, "x": 67, "y": 220}, "pitchAngle": 13.571462631225586, "rollAngle": -6.886875152587891, "yawAngle": -3.9724981784820557}] . But its value changes as we move our face then which data I can store to data base to compair and detect face for attendance?

**Comment by luicfrr** [comment]
this library is only used to detect faces not to check faces similarity. You need another sollution for this
